PayloadPkg/OsLoader/OsLoader.c

```diff
@@ -670,6 +676,27 @@ BeforeOSJump (
   }
   AddMeasurePoint (0x4100);
 
+  // Add the measurepoint timestamp to the HOB so that it can be updated in the ACPI table
+  PerfData = GetPerfDataPtr();
+  GuidHob = GetNextGuidHob (&gLoaderPerformanceInfoGuid, (VOID *)(UINTN)PcdGet32 (PcdPayloadHobList));
+  if (GuidHob != NULL) {
+    PerfInfo            = (PERFORMANCE_INFO *)GET_GUID_HOB_DATA (GuidHob);
+    PerfInfo->TimeStamp[PerfInfo->Count] = PerfData->TimeStamp[PerfData->PerfIndex-1];
+  }
+
+  // Add perf data to FPDT - SBL Performance Table
+  SystemTableInfo = GetSystemTableInfo ();
+  if ((SystemTableInfo != NULL) && (SystemTableInfo->AcpiTableBase != 0)) {
+    Rsdp = (EFI_ACPI_3_0_ROOT_SYSTEM_DESCRIPTION_POINTER *)(UINTN)SystemTableInfo->AcpiTableBase;
+    Fpdt = FindAcpiTableBySignature((EFI_ACPI_DESCRIPTION_HEADER *)(UINTN)Rsdp->RsdtAddress, SIGNATURE_32('F', 'P', 'D', 'T'), NULL);
```

Comment:
Does the similar ACPI gets populated for boot with UEFI payload?

Reply:
UEFI payload populates the Firmware Basic Boot Performance Table (FBPT) and S3 Performance Table (S3PT) tables in the FPDT. SBLT is not currently populated.